cron_v10/src/lib.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fil_actors_runtime_v10::runtime::{ActorCode, Runtime};
+use fil_actors_runtime_v10::{actor_dispatch, actor_error, ActorError, SYSTEM_ACTOR_ADDR};
+
+use fvm_ipld_encoding::tuple::*;
+use fvm_shared::econ::TokenAmount;
+
+use fvm_shared::METHOD_CONSTRUCTOR;
+use num_derive::FromPrimitive;
+use num_traits::Zero;
+
+pub use self::state::{Entry, State};
+
+mod state;
+pub mod testing;
+
+#[cfg(feature = "fil-actor")]
+fil_actors_runtime::wasm_trampoline!(Actor);
+
+// * Updated to specs-actors commit: 845089a6d2580e46055c24415a6c32ee688e5186 (v3.0.0)
+
+/// Cron actor methods available
+#[derive(FromPrimitive)]
+#[repr(u64)]
+pub enum Method {
+    Constructor = METHOD_CONSTRUCTOR,
+    EpochTick = 2,
+}
+
+/// Constructor parameters for Cron actor, contains entries
+/// of actors and methods to call on each epoch
+#[derive(Default, Debug, Serialize_tuple, Deserialize_tuple)]
+pub struct ConstructorParams {
+    /// Entries is a set of actors (and corresponding methods) to call during EpochTick.
+    pub entries: Vec<Entry>,
+}
+
+/// Cron actor
+pub struct Actor;
+
+impl Actor {
```

Comment:
We don't need to have anything connected to the `Actor`

Reply:
I see, removed

Reply:
Not completely

Reply:
updated with changes